homeassistant/components/webdav/backup.py

```diff
@@ -75,13 +75,9 @@ async def wrapper(self: WebDavBackupAgent, *args: P.args, **kwargs: P.kwargs) ->
         except UnauthorizedError as err:
             raise BackupAgentError("Authentication error") from err
         except WebDavError as err:
-            _LOGGER.error(
-                "Error during backup in %s:, message %s",
-                func.__name__,
-                err,
-            )
-            _LOGGER.debug("Full error: %s", err, exc_info=True)
```

Comment:
Side note: debug logging is always ok if you want more info than what's in the exception message.